libraries/AP_RCProtocol/AP_RCProtocol.cpp

```diff
@@ -33,6 +33,7 @@
 #include "AP_RCProtocol_FPort.h"
 #include "AP_RCProtocol_FPort2.h"
 #include "AP_RCProtocol_DroneCAN.h"
+#include "AP_RCProtocol_MavlinkRadio.h"
```

Comment:
```suggestion
#include "AP_RCProtocol_MAVLinkRadio.h"
```

Reply:
done. Also changed the class name to AP_P_RCProtocol_MAVLinkRadio.